src/System.ComponentModel.Composition/src/System.ComponentModel.Composition.csproj

```diff
@@ -6,6 +6,7 @@
     <RootNamespace>System.ComponentModel.Composition</RootNamespace>
     <AssemblyName>System.ComponentModel.Composition</AssemblyName>
     <DefineConstants>$(DefineConstants);FEATURE_TRACING</DefineConstants>
+    <RunApiCompat>false</RunApiCompat>
```

Comment:
Can you add a comment like
`<-- Workaround for issue <linkto issue you open> -->`

Reply:
and in the issue you would note that this workaround needs to be removed